src/commandargs.cpp

```diff
@@ -34,11 +47,20 @@ void ParseCommandArgs(int argc, char* argv[], ProgramConfiguration config)
         {
             CurrentState->Action(CurrentSettingOptions);
             CurrentSettingOptions.clear();
+            inFlag = true;
         }
-        else
+        else if(inFlag)
         {
+        	// Flags consume one argument
```

Comment:
Do flags only take one argument? I was under the assumption that we could pass multiple arguments under a given flag but now that I think about it... you might be right that there's only one

Reply:
This isn't law, but [convention](http://journals.ecs.soton.ac.uk/java/tutorial/java/cmdLineArgs/conventions.html). 

It wouldn't be unreasonable to pass in a list/vector/array attached to a flag for commandline parsing so:

`pebble -v sev0 sev1 sev2 foo.pebl`

This means you'd need to permit some form of manual termination of flags for parsing, though, which many systems implement as `-` or `--` as seen in the unixen [getopt(3)](https://manpage.me/?q=getopt&sektion=3&manpath=FreeBSD+12-CURRENT+and+Ports), so:

`pebble -v sev0 sev1 sev2 -- foo.pebl` 

etc. to indicate end of parse where the problem arises in the 'last flag before regular arguments'. 

I had to [do this once in a script](https://github.com/henesy/Plan9/blob/master/rc/sedi) I wrote and ended up always needing to pass `--` to get the commandline parsers to stop manually. 

The implementation and usage are both simplified by mandating one argument per flag.